src/components/notebook-utils.js

```diff
@@ -48,8 +49,16 @@ export const notebookNameValidator = existing => ({
     message: 'already exists'
   }
 })
+// removes leading dirs and a file ext suffix on paths
+export const getDisplayName = name => stripExtension(getFileName(name))
 
-export const notebookNameInput = ({ inputProps, ...props }) => h(ValidatedInput, {
+// removes all paths up to and including the last slash
+export const getFileName = name => path.basename(name)
+
+// removes a file extension from a string
+export const stripExtension = name => name.replace(/\.[^/.]+$/, '')
```

Comment:
Something like this might be simpler
```suggestion
export const getFileName = _.replace(/^.+\/.+\//, '')

// removes a file extension from a string
export const stripExtension = _.replace(/\.[^/.]+$/, '')
```

Reply:
The get file name regex didn't work for me... point taken though, removed the import and found another solution